inst/testdata/synthetic_list.yml

```diff
@@ -6,9 +6,6 @@
 - name: finalSE_combo_2dose_nonoise
   file: finalSE_combo_2dose_nonoise.RDS
   description: 
-- name: finalSE_many_drugs
-  file: finalSE_many_drugs.RDS
-  description:
```

Comment:
Shouldn't `finalSE_combo_1dose_many_drugs` and `finalSE_many_lines` also be removed - as the task description listed?

Reply:
Updated task description, thank you.

Reply:
`synthetic_list.yml` should be also updated - please, remove also description for `finalSE_combo_1dose_many_drugs` and `finalSE_many_lines` from this list.

Reply:
As requested, updated task descritpion.